examples/tx-example.py

```diff
@@ -89,13 +92,17 @@ def add_coin_to_admin():
     """
     Add 1000.00 units of 'coin#domain' to 'admin@test'
     """
+    first_time = int(round(time.time()*1000))
+    time.sleep(1)
```

Comment:
With this improvement, you reduce the number of transactions per second. This could be not what we want. It changes behavior of example much. 
I'd suggest creating another example.